Victoradukwu/More-Recipes
# #155045634 User should be able to view full details of a recipe

#### What does this PR do?
- Displays the details of a specific recipe
#### Description of Task to be completed?
- Create the RecipeDetailsPage, RecipeInfo, Ratings, Reviews, ReviewLists components
- Refactor the server side recipe controller and response handlers to return appropriate format
- Create the necessary actions and reducers
- Implement the logic on the component
- Restore the Add recipe component, too
#### How should this be manually tested?
- Start the app with the command `npm run start:dev`
- Go to localhost:8000 to see the landing page
- Click the `Check details` button on any recipe. The details will be displayed
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
 - feature/155045634/user-should-be-able-to-view-full-details-of-a-recipe

#### Screenshots (if appropriate)

#### Questions:
N/A
